app/models/incomeSourceDetails/TaxYear.scala

```diff
@@ -16,10 +16,25 @@
 
 package models.incomeSourceDetails
 
+import services.DateServiceInterface
+
 import scala.util.Try
 
 case class TaxYear(startYear: Int, endYear: Int) {
   override def toString: String = s"$startYear-$endYear"
+
+  def currentTaxYearMinusOne: TaxYear = {
```

Comment:
Please unit tests these new methods

Reply:
ah yeah, i removed the old ones and forgot to add the new ones :P